bulktest/batch_lib.inc

```diff
@@ -964,7 +964,10 @@ function prettyType($mimeType, $ext) {
 	else if ( "xml" === $ext ) {
 		return "xml";
 	}
-	else if ( false !== strpos($mimeType, "flash") || "mp4" === $ext || "swf" === $ext || "f4v" === $ext || "flv" === $ext ) {
+    //Video extensions mp4, webm, ts, m4v, m4s, m4v, mov, ogv
+	else if ( false !== strpos($mimeType, "flash") || (true === strpos($mimeType, "webm") || (true === strpos($mimeType, "mp4") || (true === strpos($mimeType, "flv"))
```

Comment:
Careful, [`strpos`](http://php.net/manual/en/function.strpos.php) never returns `true`. Let's stick with the syntax used for flash:

```php
false !== strpos($mimeType, "webm")
```

Reply:
OK. Such a pity we don't have this code in Python…